content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Kubernetes Services for Egress Traffic
+description: Shows how to configure Istio  Kubernetes External Services.
+keywords: [traffic-management,egress]
+weight: 60
+---
+
+Kubernetes [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+services and Kubernetes services with
```

Comment:
Kubernetes external name services?  This reads odd.  Kubernetes services with type ExternalName ?

Reply:
@linsun I follow the Kubernetes documentation: https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors

> An ExternalName Service is a special case of Service ...